target/linux/ramips/base-files/etc/board.d/02_network

```diff
@@ -337,6 +338,10 @@ ramips_setup_macs()
 		lan_mac=$(mtd_get_mac_binary factory 40)
 		wan_mac=$(mtd_get_mac_binary factory 46)
 		;;
+	rb750gr3)
+		wan_mac=$(cat /sys/class/net/eth0/address)
+		lan_mac=$(macaddr_add "$wan_mac" 1)
```

Comment:
It seems `mtd-mac-address-increment = <1>;` only works for incrementing WAN MAC address from LAN MAC address.

In the case of RB750Gr3, WLLLL layout, would `mtd-mac-address-increment` function as expected? Because only wan_mac was specified here.